### DISCUSSION TITLE
How to check if libebur-128 is installed in the flatpak environment

### DISCUSSION THREAD

**Discussion by Lakis01** [fooyin/fooyin#334] [Q&A] [answered]
In the changes to 0.8.0 it says "calculating ReplayGain uses libebur128 if found, or FFmpeg as a fallback". I am running Ubuntu 24.04. I can install the package via apt install libebur128-1. I think it was automatically installed when I installed the fooyin nomble .deb package. However, I thought that since flatpak is its own kind of environment, installing libebur128 via apt might not mean that a fooyin installation through flatpak does have access to it. How can I check if fooyin finds libebur128 or has already installed it as a dependency in the flatpak environment?
Calculating replaygain does work, but I cannot tell if it is done via libebur128 or ffmpeg.
Thank you!

**Comment by ludouzi** [comment, marked as answer]
All of the official packages will include libebur128 as a dependency, including the flatpak (it is built and bundled within).

The only way to tell currently is that the FFmpeg scanner will log output while it is running, while the ebur128 scanner doesn't. I'll add a log message when starting calculations for both so it's more clear for future releases.
